backend/src/main/java/gr/uom/strategicplanning/services/ExternalAnalysisService.java

```diff
@@ -20,19 +20,35 @@ public class ExternalAnalysisService {
     @Value("${services.external.codeInspector.url}")
     private String CODE_INSPECTOR_URL;
 
+    private String CODE_INSPECTOR_ANALYZE_HOTSPOTS = "/api/analysis/prioritize_hotspots";
+    private String PYASSESS_ANALYZE_REPO = "/api/analysis";
+
     public ExternalAnalysisService(RestTemplate restTemplate) {
         this.pyAssessServiceStrategy = new PyAssessServiceStrategy(restTemplate);
         this.codeInspectorServiceStrategy = new CodeInspectorServiceStrategy(restTemplate);
     }
 
+
+    // Simulate analysis (just to make sure language finding works)
+    private void fakeAnalysis(String name, int waitTime) {
```

Comment:
Shouldn't this be removed?

Reply:
I left it just in case we need again for debugging